lib/Basics/win-utils.cpp

```diff
@@ -282,11 +282,13 @@ char* TRI_GETCWD(char* buffer, int maxlen) {
   char* rc = nullptr;
   wchar_t* rcw;
   int wBufLen = maxlen;
-  wchar_t* wbuf = (wchar_t*)malloc(wBufLen * sizeof(wchar_t));
-  if (wbuf == nullptr) {
+  // wchar_t* wbuf = (wchar_t*)malloc(wBufLen * sizeof(wchar_t));
+  auto wbuf = std::make_unique<wchar_t[]>(wBufLen);
+
+  if (wbuf.get() == nullptr) {
```

Comment:
`wbuf` cannot become a nullptr here.
if `make_unique` fails, it will throw an exception.